test/client/fixtures/sandbox/iframe-flag-test.js

```diff
@@ -0,0 +1,299 @@
+var urlUtils = hammerhead.get('./utils/url');
+
+var iframeSandbox = hammerhead.sandbox.iframe;
+var nativeMethods = hammerhead.nativeMethods;
+var browserUtils  = hammerhead.utils.browser;
+
+QUnit.testStart(function () {
+    // NOTE: The 'window.open' method used in QUnit.
+    window.open       = nativeMethods.windowOpen;
+    window.setTimeout = nativeMethods.setTimeout;
+    iframeSandbox.on(iframeSandbox.RUN_TASK_SCRIPT, initIframeTestHandler);
+    iframeSandbox.off(iframeSandbox.RUN_TASK_SCRIPT, iframeSandbox.iframeReadyToInitHandler);
+});
+
+QUnit.testDone(function () {
+    iframeSandbox.off(iframeSandbox.RUN_TASK_SCRIPT, initIframeTestHandler);
+});
+
+function hasIframeFlag (url) {
+    var resourceType = urlUtils.parseProxyUrl(url).resourceType;
+
+    return !!resourceType && resourceType.indexOf('i') !== -1;
```

Comment:
Why do we need double negative - `!!resourceType`? 
Use `return resourceType && resourceType.indexOf('i') !== -1`